.husky/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
should it be included on this commit?

Reply:
This is added because I'm merging master into this branch.